hero2vec.py

```diff
@@ -0,0 +1,162 @@
+#PyTorch libraries
+import torch
+import torch.nn as nn
+import torch.optim as optim
+
+#Custom Python files
+from dataloader import Hero2vecDataset, split_dataloader
+
+#Other libraries
+import numpy as np
+
+class Hero2vecNetwork(nn.Module):
+    '''
+    Hero2vec network.
+    '''
+    
+    def __init__(self, embedding_dim, heropool_size=129):
+        '''
+        Inputs:
+            embedding_dim: int. The number of embedding dimension.
+        Keywords:
+            heropool_size: int. The number of largest hero index. (Not the number of total heroes, since some index are not used currently in Dota2.)
+        '''
+        super().__init__()
+        self.embedding_dim = embedding_dim
+        self.hero_embedding = nn.Embedding(heropool_size, self.embedding_dim)
+        self.embedding_to_target = nn.Linear(self.embedding_dim, heropool_size)
+        self.init_network()
+        
+    def init_network(self):
+        '''
+        Initiallize the network.
+        Currently not used. Maybe will need it later.
+        '''
+        pass
+    
+    def forward(self, inputs):
+        '''
+        Inputs:
+            inputs: torch.LongTensor, size=(4,)
+        Returns:
+            out: torch.LongTensor, size=(1,)
+        '''
```

Comment:
Understandably this is just copied over from a separate repo. But I think this area could use some more docstrings. 

Recommend:
* Add docstring about what the goal of Hero2Vec is doing.
* What the forward return should be (i.e. an embedding of a Dota Hero into some N-dimensional space).

Reply:
Oh, again: you can save that for later.